Update elements reference - Closes #552 (#570)
LiskHQ/lisk-docs: modules/ROOT/pages/reference/lisk-elements/transactions.adoc

      payloadLength: 0,
      previousBlockId: '9326981395427095175',
      reward: '500000000',
      seedReveal: 'abe2a66d7a35fd7b580e977d9f7911ae',
      timestamp: 122329567,
      totalAmount: '0',
      totalFee: '0',
      version: 2
    },
    reward: 0n
  }
}

  */
----

== Signing transactions

After the transaction object was created like described in the section <<create,Creating transactions>>, it is needed to sign the transaction before it can be broadcasted to a node.

[source,js]
----
tx.sign(
    '7158c297294a540bc9ac6e474529c3da38d03ece056e3fa2d98141e6ec54132d', <1>
    'account reform outdoor curtain animal zoo best gain super glue bacon endless' <2>
    );
/*
TransferTransaction {
  senderPublicKey: 'f7425ba1b192e07639a0304531e21117ccc1852279b6ec7c296b18bd95bcc4c3',
  signatures: [
    'f6635a1f53f1e97443771c2b31b9f8cdfb1a5afb875f5d2bae8fec71c861fc8f3c9c312ac4fbc8a404a7b906b1bc7d62ca0851568ee30425452a497ab37caa0b'
  ],
  nonce: 0n,
  fee: 0n,
  type: 8,
  _id: '10457170653864555613',
  confirmations: undefined,
  blockId: undefined,
  height: undefined,
  receivedAt: undefined,
  asset: { data: 'Hello Lisk!', recipientId: '123L', amount: 15000000n }
}
 */
----

> <1> The network identifier of the network, you want to post the transaction to.

Reviewer: ```suggestion
<1> The network identifier of the network, where you wish to post the transaction.
```

<2> Enter here the 12 word mnemonic passphrase for an account

After the transaction is signed, the `signatures`, `senderPublicKey` and `networkIdentifier` are added to the transaction object.

> It is also possible to get the minimum fee for this particular trasnaction, to be accepted by the network:

Reviewer: ```suggestion
It is also possible to get the minimum fee for this particular transaction, to be accepted by the network:
```

`undefined`: if the public key is verified.
 A `TransactionError` is thrown, if the verification fails.

=== verifyMinRemainingBalance

> Verifies if an account has enough remaining balance in the account.

Reviewer: ```suggestion
Verifies if the remaining balance in the account is sufficient.
```
